packages/flutterfire_starter/__brick__/lib/src/sample_feature/sample_item_details_view.dart

```diff
@@ -2,15 +2,16 @@ import 'package:flutter/material.dart';
 
 /// Displays detailed information about a SampleItem.
 class SampleItemDetailsView extends StatelessWidget {
-  const SampleItemDetailsView({Key? key}) : super(key: key);
+  const SampleItemDetailsView({Key? key, this.itemId}) : super(key: key);
 
-  static const routeName = '/sample_item';
+  static const routeName = '{{^analyticswithgorouter}}/{{/analyticswithgorouter}}sample_item';
```

Comment:
Just checking if there should be a forward slash before `sample_item`? `{{/analyticswithgorouter}}/sample_item`

Reply:
When using GoRouter you have to remove the forward slash, so it's okay this way